app/models/ca_bundle_displays.php

```diff
@@ -2277,6 +2327,17 @@ public function getDisplayValue($po_result, $pn_placement_id, $options=null) {
 		if(!($vs_template = trim($options['template'])) && (sizeof($va_bundle_bits) == 1) && ($t_instance = Datamodel::getInstanceByTableName($va_bundle_bits[0], true))) {
 			$vs_template = $this->getAppConfig()->get($va_bundle_bits[0]."_default_bundle_display_template");
 		}
+
+		if (preg_match('/^custom_display_component/', $va_bundle_bits[1])) {
+			$vs_component = str_replace('custom_display_component_', '', $va_bundle_bits[1]);
+			$vs_filename = __CA_APP_DIR__ . "/models/CustomDisplayComponents/{$va_bundle_bits[0]}/{$vs_component}.php";
+			if (file_exists($vs_filename)) {
+				$vs_template = file_get_contents($vs_filename);
+			} else {
+				$vs_template = 'ERROR - missing template ' . $vs_filename;
+			}
+			error_log($vs_filename);
```

Comment:
move log into else statement and include something like 'Failed to find missing template'

Reply:
this is actually just debugging that I missed >.<